Luke7787/ImmaculateInventors
# Cloud

Summary: 
- Moved to atlas
- removed conns

How to test:
npm run dev

Test in boomerang manually with the same links as before.

Go to the cloud and see if it is updated in the cloud:
https://cloud.mongodb.com/v2/660db89249bbfd7ebb48969f#/metrics/replicaSet/660dbacb76b3a13ad91c6318/explorer/myInventory/users/find
